connectors/utils.py

```diff
@@ -518,20 +518,35 @@ def is_expired(expires_at):
     return datetime.utcnow() >= expires_at
 
 
-def get_pem_format(key, max_split=-1):
+def get_pem_format(
+    key,
+    postfix="-----END CERTIFICATE-----",
+    max_split=-1,
+):
     """Convert key into PEM format.
 
     Args:
         key (str): Key in raw format.
+        postfix (str): Certificate end string.
         max_split (int): Specifies how many splits to do. Defaults to -1.
```

Comment:
This parameter has a very confusing description. What does it do? Can you improve the description?

Reply:
I'm afraid I still don't understand what a "split" is